lib/defaults.js

```diff
@@ -2,260 +2,111 @@
-    //'no-octal-escape': 2,            // http://eslint.org/docs/rules/no-octal-escape
-    'no-param-reassign': 0,          // http://eslint.org/docs/rules/no-param-reassign
-    //'no-proto': 2,                   // http://eslint.org/docs/rules/no-proto
-    //'no-redeclare': 2,               // http://eslint.org/docs/rules/no-redeclare
-    //'no-return-assign': 2,           // http://eslint.org/docs/rules/no-return-assign
-    //'no-script-url': 2,              // http://eslint.org/docs/rules/no-script-url
-    //'no-self-compare': 2,            // http://eslint.or//g/docs/rules/no-self-compare
-    //'no-sequences': 2,               // http://eslint.org/docs/rules/no-sequences
-    //'no-throw-literal': 2,           // http://eslint.org/docs/rules/no-throw-literal
-    //'no-with': 2,                    // http://eslint.org/docs/rules/no-with
-    //'radix': 2,                      // http://eslint.org/docs/rules/radix
-    'vars-on-top': 0,                // http://eslint.org/docs/rules/vars-on-top
-    'wrap-iife': [2, 'any'],         // http://eslint.org/docs/rules/wrap-iife
-    //'yoda': 2,                       // http://eslint.org/docs/rules/yoda
+    'consistent-return': [1, { treatUndefinedAsUnspecified: true}],          
+    'eqeqeq': 2,                     
+    'no-eq-null': 2,                 
+    'no-param-reassign': 0,          
+    'vars-on-top': 0,                
+    'wrap-iife': [2, 'any'],         
 
     /**
      * Style
      */
-    /*'brace-style': [                 // http://eslint.org/docs/rules/brace-style
-      2,
-      '1tbs',
-      { 'allowSingleLine': true }
-    ],*/
-    'camelcase': [2, {               // http://eslint.org/docs/rules/camelcase
+    'camelcase': [2, {               
       'properties': 'always'
     }],
-    /*'comma-spacing': [2, {           // http://eslint.org/docs/rules/comma-spacing
-      'before': false,
-      'after': true
-    }],*/
-    //'comma-style': [2, 'last'],      // http://eslint.org/docs/rules/comma-style
-    'eol-last': 0,                   // http://eslint.org/docs/rules/eol-last
-    'func-names': 0,                 // http://eslint.org/docs/rules/func-names
-    /*'indent': [                      //http://eslint.org/docs/rules/indent
-      2,
-      2,
-      { 'SwitchCase': 1 }
-    ],*/
-    /*'key-spacing': [2, {             // http://eslint.org/docs/rules/key-spacing
-        'beforeColon': false,
-        'afterColon': true
-     }],*/
+    'eol-last': 0,                   
+    'func-names': 0,                 
     'max-len': [2, 200, 2, {
       ignoreUrls: true,
       ignoreComments: false
     }],
     'max-statements': [1, 30],
-    /*'new-cap': [2, {                 // http://eslint.org/docs/rules/new-cap
-      'newIsCap': true
-    }],*/
-    'no-multiple-empty-lines': [2, { // http://eslint.org/docs/rules/no-multiple-empty-lines
+    'no-multiple-empty-lines': [1, { 
```

Comment:
I recognize that it's just a stylistic thing. I prefer it, just like I prefer error on trailing whitespace, but I don't think I can provide a cogent argument as to why **any** style rules should error instead of warn.